canvas_site_deletion/views.py

```diff
@@ -161,3 +174,33 @@ def delete(request, pk):
     messages.success(request, f"Successfully deleted Canvas course {canvas_course_id} and cleaned up course_instance {pk}")
 
     return render(request, 'canvas_site_deletion/index.html')
+
+
+def get_account_hierarchy(course_id) -> list:
+    """
+    Get the list of accounts associated with a given course ID
+    """
+    account_ids = []
+
+    # use course id to get account id
+    canvas_course = canvas_get_course(SDK_CONTEXT, course_id).json()
+    canvas_course_account_id = canvas_course["account_id"] 
+    account_ids.append(canvas_course_account_id)
+
+    # traverse account hierarchy in both directions
+    # first downwards to find sub-accounts
+    sub_accounts = get_sub_accounts_of_account(SDK_CONTEXT, canvas_course_account_id).json()
```

Comment:
There is a `recursive` argument for this call:
`If true, the entire account tree underneath this account will be returned (though still paginated). If false, only direct sub-accounts of this account will be returned. Defaults to false.`